package.json

```diff
@@ -78,6 +78,6 @@
   "packageManager": "pnpm@8.6.5",
   "engines": {
     "node": "20.x",
-    "pnpm": ">=8"
+    "pnpm": "9.x"
```

Comment:
Sample: https://vercel.com/docs/errors/error-list#pnpm-engine-unsupported

Reply:
Do I need to specify the exact version that I used locally to update `pnpm-lock.yaml` ?